topic_browser_plugin/topic_browser_data.proto

```
@@ -0,0 +1,242 @@
+// ────────────
+//    ┌───────────┐   UnsBundle  (LZ4 + hex)   ┌─────────────┐
+//    │ tag_proc. │ ───────────────────────────►│   FSM       │
+//    └───────────┘                             └─────────────┘
+//      UnsBundle.events[*] : EventTableEntry         (~ring-100)
+//      UnsBundle.uns_map   : TopicInfo (deltas only)
+//
+// ---------------------------------------------------------------------------
+
+syntax = "proto3";
+
+package umh.events;
```

Buf (1.54.0)

Files with package "umh.events" must be within a directory "umh/events" relative to root but were in directory "topic_browser_plugin".

(PACKAGE_DIRECTORY_MATCH)